framework/test_context.py

```diff
@@ -18,6 +18,14 @@ def test_fcc_cms(self):
         env = dict(PODIO='.', FCCEDM='.', FCCPHYSICS='.', HEPPY='.',
```

Comment:
Hi Colin! A lot of these environment variables aren't actually set anymore when setting up FCC software as we try to do more standardized installs. I think currently there is only `FCCVIEW` set when using the externals from cvmfs. But I guess it would also be reasonable to ask the user in which context to run, with a command line argument maybe?